### DISCUSSION TITLE
Non-random cluster_name for examples/kubernetes

### DISCUSSION THREAD

**Discussion by kriswuollett** [digitalocean/terraform-provider-digitalocean#690] [Q&A]
Is there a particular reason why `cluster_name` being `tf-k8s-${random_id.cluster_name.hex}` was needed in #564? When I remove the `random_id` resource and just set `cluster_name` to `my_cluster`, the output of `terraform plan` has the error:

```
│ Error: Unable to find cluster with name: my-cluster
│ 
│   with module.kubernetes-config.data.digitalocean_kubernetes_cluster.primary,
│   on kubernetes-config/main.tf line 18, in data "digitalocean_kubernetes_cluster" "primary":
│   18: data "digitalocean_kubernetes_cluster" "primary" {
```

I'm new to Terraform so would like to learn more about this issue if it is not technically a bug. Thanks!

**Comment by scotchneat** [comment]
Hi @kriswuollett, thanks for the question. I was able to reproduce the behavior you described but will need to look into it more before I can suggest a solution.

**Reply by kriswuollett** [reply]
OK thanks. I think I was able to find what seems like a way in #695. Just don't know at the moment how the randomized name helps currently.
